Resources/Prototypes/Traits/disabilities.yml

```diff
@@ -222,7 +224,7 @@
 - type: trait
   id: SnailPaced
   category: Physical
-  points: 5
+  points: 8
```

Comment:
5 and 8 points for these is iffy. It's true that speed matters a lot in combat, but outside of combat those traits barely affect anything.